src/Ilios/ApiBundle/Resources/swagger/definitions/Userevent.yml

```diff
@@ -0,0 +1,4 @@
+Userevent:
```

Comment:
All properties are inherited from Calendarevent so they will pick up that status from there.